models/jules/R/model2netcdf.JULES.R

```diff
@@ -32,19 +32,19 @@ model2netcdf.JULES <- function(outdir) {
                 row.names = TRUE,
                 quote = FALSE)
     
-    vars <- ncdf4.helpers::nc.get.variable.list(nc)
+    vars <- names(nc[['var']])
     # Check that frac is reported
     if("frac_grid" %in% vars){
       frac <- ncdf4::ncvar_get(nc, "frac_grid")
     } else {
       PEcAn.logger::logger.warn("Surface type fraction is not an output and thus other outputs may not be parseable")
     }
     
-    base_dims <- ncdf4.helpers::nc.get.dim.names(nc, "GPP") # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
+    base_dims <- nc$var[["GPP"]]$dim[[1]]$name # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
```

Comment:
Unless I'm missing something, this doesn't give quite the same result -- `nc.get.dim.names` returns the names of all the dimensions (for most of our files, probably `c("latitude", "longitude", "time")`), not just the first one. Maybe we want something like ```vapply(nc$var[["GPP"]]$dim, `[[`, character(1), "name")```?

Reply:
Lol, @ashiklom beat me to the identical suggestion